test/multiapi/Expected/AcceptanceTests/MultiapiWithSubmodule/multiapiwithsubmodule/submodule/v2/aio/_configuration.py

```diff
@@ -57,4 +57,4 @@ def _configure(
         self.redirect_policy = kwargs.get('redirect_policy') or policies.AsyncRedirectPolicy(**kwargs)
         self.authentication_policy = kwargs.get('authentication_policy')
         if self.credential and not self.authentication_policy:
-            self.authentication_policy = policies.AsyncBearerTokenCredentialPolicy(self.credential, *self.credential_scopes, **kwargs)
+            self.authentication_policy = AsyncARMChallengeAuthenticationPolicy(self.credential, *self.credential_scopes, **kwargs)
```

Comment:
I would also add an `AcceptanceTests` for this change

Reply:
I can't find related example about `AsyncARMChallengeAuthenticationPolicy`. How to add test for it? Is there an example?

Reply:
add some test : 
test_credential_authentication_policy_default
test_arm_authentication_policy_default
test_authentication_policy_default
test_arm__authentication_policy_default
test_arm_http_authentication_policy_default_default

Reply:
I could pass the test but it seems that CI failed because of other reasons.

Reply:
yeah, the CI keeps hanging, i've been working on it all day, hopefulyl it works by the time you start working